test/app_engine_test_utils.py

```diff
@@ -0,0 +1,25 @@
+from google.appengine.ext import testbed
```

Comment:
alternative to test utils like this is to base class the tests which can lead to unexpected behavior in my experience, unless you do things like multiple inheritance which is silly

Reply:
yeah this way is good